bin/deployment/gen_certs.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+SCRIPT_NAME=$(basename "$0")
+
+# This works fine for our dev deployment -- especially within Vagrant images -- but may not work as
+# expected on systems with multiple interfaces or multiple external IPs (i.e. IPv4 and IPv6). On
+# such systems, this script should be run with explicit arguments for these values.
```

Comment:
Restart your system, rearrange the interfaces virtually, or otherwise do something that would cause the output to not be predictable. It should still function, but it may not pick the desired IP.

Reply:
Ah, I see, so the problem is it just gets the first IP instead of the specific external IP that someone wants